ansible_events_ui/ruleset.py

```diff
@@ -56,57 +70,85 @@ async def activate_rulesets(
     Call ansible-events with ruleset, inventory, and extravars added
     as volumes to a container.
     """
-    tempdir = tempfile.mkdtemp(prefix="ruleset_manager")
-    result = await db.execute(select(playbooks))
-    for playbook in result.all():
-        with open(os.path.join(tempdir, playbook.name), "w") as f:
-            f.write(playbook.playbook)
-    rules_file = os.path.join(tempdir, "rules.yml")
-    with open(rules_file, "w") as f:
-        f.write(rulesets)
-    inventory_file = os.path.join(tempdir, "inventory.yml")
-    with open(inventory_file, "w") as f:
-        f.write(inventory)
-    vars_file = os.path.join(tempdir, "vars.yml")
-    with open(vars_file, "w") as f:
-        f.write(extravars)
+    settings = load_settings()
+    local_working_directory = working_directory
+    ensure_directory(local_working_directory)
+
+    if settings.deployment_type == "local":
```

Comment:
Better to use enum for fixed set of string values.

Reply:
Please add a PR for this after this merges.

Reply:
Can you please add a `TODO` comment then.